.ci/setup_env_opensuse.sh

```diff
@@ -15,6 +15,12 @@ source "${cidir}/lib.sh"
 echo "Remove openSUSE cloud repo"
 sudo zypper rr openSUSE-Leap-Cloud-Tools
 
+echo "Add filesystems repo"
+filesystem_repo="https://download.opensuse.org/repositories/filesystems/openSUSE_Leap_15.1/filesystems.repo"
```

Comment:
Can you use a variable from `os-release` maybe rather than hard-coding `15.1` here? I'm guessing the answer is no as you have used a variable below. If you can't use a variable, it might be worth making this clearer somehow (adding a comment and/or creating your own variable (`hardcoded_version=15.1` or something?) to make this stand out more.